### PR TITLE
subversion: Depends on apr and apr-util

### LINKED ISSUE #408: subversion: error: Serf was explicitly enabled but an appropriate version was not found
https://gist.github.com/anonymous/489861971479f5bf9966

```
==> Downloading https://www.apache.org/dyn/closer.cgi?path=apr/apr-1.5.2.tar.bz2
==> Best Mirror http://apache.mirrors.tds.net/apr/apr-1.5.2.tar.bz2
######################################################################## 100.0%
==> ./configure --prefix=/local/mnt/workspace/naseer/.linuxbrew/Cellar/apr/1.5.2/libexec
==> make install
/local/mnt/workspace/naseer/.linuxbrew/Cellar/apr/1.5.2: 54 files, 1.5M, built in 41 seconds
naseer:$D2/shamu-pdk/kernel/drivers/video/msm/mdss $ brew install subversion
==> Downloading https://www.apache.org/dyn/closer.cgi?path=subversion/subversion-1.8.13.tar.bz2
Already downloaded: /local/mnt/workspace/naseer/.linuxbrew/cache/subversion-1.8.13.tar.bz2
==> Patching
patching file configure
Hunk #1 succeeded at 25386 (offset 20 lines).
patching file subversion/bindings/swig/perl/native/Makefile.PL.in
==> Downloading https://serf.googlecode.com/svn/src_releases/serf-1.3.8.tar.bz2
Already downloaded: /local/mnt/workspace/naseer/.linuxbrew/cache/subversion--serf-1.3.8.tar.bz2
==> /local/mnt/workspace/naseer/.linuxbrew/opt/scons/bin/scons PREFIX=/local/mnt/workspace/naseer/.linuxbrew/Cellar/subversion/1.8.13/libexec/serf GSSAPI=/usr CC=/local/mnt/workspace/naseer/.linuxbrew/bin/gcc-4.9 CFLAGS=-Os -w -pipe -march=
    env.ParseConfig('$APU --ldflags --includes --link-ld --libs')
  File "/local/mnt/workspace/naseer/.linuxbrew/Cellar/scons/2.3.4/libexec/scons-local/SCons/Environment.py", line 1579:
    return function(self, self.backtick(command))
  File "/local/mnt/workspace/naseer/.linuxbrew/Cellar/scons/2.3.4/libexec/scons-local/SCons/Environment.py", line 597:
    raise OSError("'%s' exited %d" % (command, status))
```

### PR DESCRIPTION
**Pull request by naseer**
Fixes #408

### CODE REVIEW

**Comment by sjackman** [comment]
I haven't found that `ENV.deparallelize` was necessary. What error message did you see?

File: Library/Formula/subversion.rb [hunk 1]
```diff
@@ -70,6 +74,7 @@ class Subversion < Formula
   end
 
   def install
+    ENV.deparallelize
```
Comment (sjackman) [inline Library/Formula/subversion.rb]: Please add a comment indicating the error message that this line fixes.

File: Library/Formula/subversion.rb [hunk 2]
```diff
@@ -45,6 +45,10 @@ class Subversion < Formula
   # For Serf
   depends_on "scons" => :build
   depends_on "openssl"
+  unless OS.mac?
+      depends_on "apr"
+      depends_on "apr-util"
```
Comment (sjackman) [inline Library/Formula/subversion.rb]: Indent should be two spaces.

**Comment by sjackman** [comment]
Thanks for the patch!

**Comment by naseer** [comment]
Will address these comments, but the issue was actually not resolved properly with this on another colleagues machine. In his case, svn did install but it still complained after installation that it could not find the right version of libserf. We worked around it by removing the brew svn version from the path and using the Ubuntu 12.04 version. We only needed svn to satisfy clang-format. I will post logs when I get to a computer.

File: Library/Formula/subversion.rb [hunk 3]
```diff
@@ -70,6 +74,9 @@ class Subversion < Formula
   end
 
   def install
+    ENV.deparallelize
+    #Fixes: libtool:   error: error: relink 'libsvn_ra_serf-1.la' with the above command before installing it
+    #Please see issue https://github.com/Homebrew/linuxbrew/issues/408 for details
```
Comment (sjackman) [inline Library/Formula/subversion.rb]: Please add a space between the `#` and the comment. `brew audit --style subversion` should report this style issue.

Comment (sjackman) [inline Library/Formula/subversion.rb]: Please wrap to 80 columns.

Comment (sjackman) [inline Library/Formula/subversion.rb]: Please squash multiple consecutive spaces like `libtool:   error:` to a single space.

File: Library/Formula/subversion.rb [hunk 4]
```diff
@@ -70,7 +74,11 @@ class Subversion < Formula
   end
 
   def install
-    serf_prefix = libexec+"serf"
+    # Fixes: "libtool: error: error: relink 'libsvn_ra_serf-1.la'
+    # with the above command before installing it"
+    # Please see https://github.com/Homebrew/linuxbrew/issues/408 for details
+    ENV.deparallelize
+    serf_prefix = OS.mac? ? libexec+"serf" : prefix
```
Comment (sjackman) [inline Library/Formula/subversion.rb]: Please include a comment above this line explaining why this fix is necessary. Simply quoting the error message is good enough.

Comment (naseer) [inline Library/Formula/subversion.rb]: Done

File: Library/Formula/subversion.rb [hunk 5]
```diff
@@ -70,7 +74,12 @@ class Subversion < Formula
   end
 
   def install
-    serf_prefix = libexec+"serf"
+    # Fixes: "libtool: error: error: relink 'libsvn_ra_serf-1.la'
+    # with the above command before installing it"
+    # Please see https://github.com/Homebrew/linuxbrew/issues/408 for details
+    ENV.deparallelize
```
Comment (sjackman) [inline Library/Formula/subversion.rb]: If you have access to a Mac OS machine, can you attempt to duplicate this bug on Mac OS? If it duplicates, can you send this chunk upstream to Homebrew/homebrew?

**Comment by sjackman** [comment]
Thanks for your contribution, Naseer!

**Comment by naseer** [comment]
Thanks for helping me through the patch, Shawn :)

File: Library/Formula/subversion.rb (hunk 5 above)
Comment (naseer) [inline Library/Formula/subversion.rb]: I couldn't reproduce this issue on OS X

**Comment by sjackman** [comment]
No worries. Happy to help.

**Pull request closed without merging**